_posts/en/newsletters/2019-12-18-newsletter.md

```diff
@@ -0,0 +1,179 @@
+bech32 error detection reliability, summarizes updates to the proposed
+`OP_CHECKTEMPLATEVERIFY` opcode, and links to a discussion about the impact
+of eclipse attacks on LN channels.  Also included is our regular section
+about notable changes to popular Bitcoin infrastructure projects. <!--
+FIXME: also mention other sections if schmidty decides to write them -->
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+- **Upgrade to LND 0.8.2-beta:** this [release][lnd 0.8.2-beta] contains
+  several bug fixes and minor UX improvements, most notably for the
+  recovery of static channel backups.
+
+- **Help test C-Lightning 0.8.0 RCs:** the [release candidate][cl 0.8.0]
+  for the next version of C-Lightning switches the default network to
+  mainnet instead of testnet (see [Newsletter #75][news75 cl mainnet])
+  and adds support for basic multipath payments (described below) among
+  many other features and bug fixes.
+
+## News
+
+- **LN implementations add multipath payment support:** after a
+  significant amount of discussion and development, all three LN
+  implementations tracked by Optech have now added basic support for
+  [multipath payments][topic multipath payments].  A multipath payment
+  consists of multiple LN payments routed over different paths which may
+  all be claimed at the same time by the receiver.  This provides a
+  major upgrade to the usability of LN by allowing users with their
+  balance split across multiple channels to spend or receive funds in
+  multiple channels in the same overall payment.  With this upgrade,
+  normal non-routing users don't need to worry about how much balance or
+  capacity they have in any particular channel---they can spend or
+  receive up to their full available balance or capacity (subject to
+  other limits of their software or the protocol).
+
+- **Analysis of bech32 error detection:** Pieter Wuille sent an
+  [email][wuille bech32 post] to the Bitcoin-Dev mailing list following
+  up on the bech32 malleability concerns described in previous
+  newsletters ([#72][news72 bech32], [#74][news74 bech32], and
+  [#76][news76 bech32]) where it's possible to add or remove a `q`
+  character immediately before a `p` character at the end of a bech32
+  string.  Wuille's [analysis][wuille bech32 analysis] shows that this
+  is the only exception to bech32's expected error detection properties
+  and that "changing one constant in bech32 would resolve this issue."
+
+    Wuille plans to amend [BIP173][] to describe the weakness, propose a
+    change to limit existing bech32 address uses to either 20 byte or 32
+    byte witness program payloads, and define a modified version of
+    bech32 with the alternative constant for non-Bitcoin uses and for a
+    potential future where we want witness programs that are not 20
+    bytes or 32 bytes.
+
```

Comment:
I was thinking we'd want to mention this in the Action items section as we'd love to hear feedback from Optech members and newsletter readers on Pieter's proposal to handle this situation. Thoughts?